### PR TITLE
Add capabilities to driftscan

### PR DESCRIPTION
**Pull request by cahofer**
Summary:

- changes to `manager.py` and `beamtransfer.py`:  allow skipping the inversion of svd beams 
- changes to `psestimation.py`: summation over local m fisher matrices and followed by allreduce.
- changes to `telescope.py`: allow frequency binning like in CHIME

### CODE REVIEW

**Comment by cahofer** [comment]
These are some changes that I thought were useful to driftscan and used this code for the past months.

File: drift/core/telescope.py [hunk 1]
```diff
@@ -171,6 +179,9 @@ class TransitTelescope(with_metaclass(abc.ABCMeta, config.Reader, ctime.Observer
     freq_lower = config.Property(proptype=float, default=400.0)
     freq_upper = config.Property(proptype=float, default=800.0)
     num_freq = config.Property(proptype=int, default=50)
+    use_chime_freq = config.Property(proptype=bool, default=True)
```
Comment (jrs65) [inline drift/core/telescope.py]: This doesn't need to be quite so specific. You can mostly do this with the parameters you have in here by adding a `freq_mode` parameter like exists in `cora-makesky`

File: drift/core/psestimation.py [hunk 2]
```diff
@@ -495,18 +495,29 @@ def generate(self, regen=False):
         self.genbands()
 
         # Use parallel map to distribute Fisher calculation
-        fisher_bias = mpiutil.parallel_map(
-            self.fisher_bias_m, list(range(self.telescope.mmax + 1))
-        )
+        # fisher_bias = mpiutil.parallel_map(self.fisher_bias_m, range(self.telescope.mmax + 1))
```
Comment (jrs65) [inline drift/core/psestimation.py]: Remove old code.

**Comment by jrs65** [comment]
I've reworked this changeset and fixed up my final comments. The new version if in #84 . Closing this PR in favour of that one.

**Pull request closed without merging**